integration-tests/pom.xml

```diff
@@ -123,7 +123,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>2.15.0</version>
+            <version>2.15.2</version>
         </dependency>
```

Comment:
I guess we can delete the `jackson-annotations` dependency entirely from here.

Reply:
That's what I thought as well, but the test containers need it to run.